### PR TITLE
Test for conversation request offer

### PR DESCRIPTION
**Pull request by KashyretsIvanna**
testing:
- RequestService/Controller
- InviteService/Controller
- Dto
without e2e tests yet
зображення
зображення

### CODE REVIEW

File: src/modules/requests/mockData/mockUserData.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  hourly_rate: 20,
+  status: 'Accepted',
+};
+
+export const PostRequestMock = {
+  type: RequestType.PROPOSAL,
+  hourly_rate: 15,
+  reject: false,
+  cover_letter: 'Your cover letter with skills description',
+};
```
Comment (pavlo-chernykh) [inline src/modules/requests/mockData/mockUserData.ts:13]: do we really need another mock user data?  (src/common/constants/mockdata.ts or src/common/mocks/users.ts)

File: src/modules/requests/mockData/requestData.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    } as Category,
+    duration: 4,
+    duration_amount: DurationAmount.MONTH,
+    status: JobStatus.FINISHED,
+    skills: [],
+    conversations: [],
+    requests: [],
+    offers: [],
+  },
+};
```
Comment (pavlo-chernykh) [inline src/modules/requests/mockData/requestData.ts]: same (check comment above)

File: src/modules/requests/mockData/mockUserData.ts (hunk 1 above)
Comment (KashyretsIvanna) [reply src/modules/requests/mockData/mockUserData.ts:13]: deleted this mock

File: src/modules/requests/mockData/requestData.ts (hunk 2 above)
Comment (KashyretsIvanna) [reply src/modules/requests/mockData/requestData.ts]: fixed

**Review by pavlo-chernykh — APPROVED** [review]
approved

File: src/modules/conversations/conversation.service.spec.ts [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  });
+
+  describe('checkChatAvailability', () => {
+    it('conversations found: should call checkChatAvailability , should get response object with freelancer and array of Conversations', async (): Promise<void> => {
+      const freelancerId = 3 as number;
+      const user = { id: 1 } as User;
+      const freelancer = { id: freelancerId } as User;
+      const response = {
+        freelancer: { id: freelancerId } as User,
+        data: [{ id: 35 } as Conversation],
```
Comment (lyudad) [inline src/modules/conversations/conversation.service.spec.ts]: все as убрать

**Review by lyudad — APPROVED** [review]

**Pull request merged**